samples/dependency-injection/extensions-dependency-injection/Extensions.DependencyInjection_1/Sample/Program.cs

```diff
@@ -12,24 +12,31 @@ static async Task Main()
         #region ContainerConfiguration
 
         var endpointConfiguration = new EndpointConfiguration("Sample");
+        endpointConfiguration.UseTransport<LearningTransport>();
+
+        var serviceCollection = new ServiceCollection();
+        serviceCollection.AddSingleton<MyService>();
```

Comment:
what is MyService used for?

Reply:
It was already there, injected into the handler